src/vivarium_public_health/disease/transition.py

```diff
@@ -48,19 +79,16 @@ def __init__(
 
     # noinspection PyAttributeOutsideInit
     def setup(self, builder: Builder) -> None:
-        rate_data, pipeline_name = self.load_transition_rate_data(builder)
-        self.base_rate = builder.lookup.build_table(
-            rate_data, key_columns=["sex"], parameter_columns=["age", "year"]
-        )
+        lookup_columns = get_lookup_columns([self.lookup_tables["transition_rate"]])
         self.transition_rate = builder.value.register_rate_producer(
-            pipeline_name,
+            self.transition_rate_pipeline_name,
             source=self.compute_transition_rate,
-            requires_columns=["age", "sex", "alive"],
-            requires_values=[f"{pipeline_name}.paf"],
+            requires_columns=lookup_columns + ["alive"],
```

Comment:
Why is 'alive' not just part of the `get_lookup_columns` return?

Reply:
'alive' is not a column that the LookupTable uses. It is needed by the pipeline because the pipeline source uses it separately to filter the data.